src/java.desktop/macosx/native/libawt_lwawt/awt/CPrinterJob.m

```diff
@@ -496,6 +527,17 @@ static void javaPrinterJobToNSPrintInfo(JNIEnv* env, jobject srcPrinterJob, jobj
     } else {
        [dst setJobDisposition:NSPrintSpoolJob];
     }
+
+    int sides = (*env)->CallIntMethod(env, srcPrinterJob, jm_getSides);
```

Comment:
it seems the code around uses the "java" types like jint.
>jint copies = (*env)->CallIntMethod(env, srcPrinterJob, jm_getCopies);

Reply:
Could you review the updated fix?

`int` type is changed to `jint` for sides variable.